webrtc.html

```diff
@@ -5969,8 +5969,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             "idlMemberType">sequence&lt;<a>RTCRtpCodecCapability</a>&gt;</span></dt>
             <dd>
               <p>Supported media codecs as well as entries for RTX, RED and FEC
-              mechanisms. There will only be a single entry in
-              <code>codecs[]</code> for retransmission via RTX.</p>
+              mechanisms. For retransmission via RTX, there will be an entry
+              within <code>codecs[]</code> for each codec being retransmitted,
+              with <code>sdpFmtpLine</code> containing the parameters.</p>
```

Comment:
@aboba Is there a reason it should be an empty string rather than undefined? Not that it really matters, just wondering.